vector/src/main/java/im/vector/fragments/VectorMessageListFragment.java

```diff
@@ -433,7 +433,7 @@ public void onE2eIconClick(final Event event, final MXDeviceInfo deviceInfo) {
         if (null != event.getCryptoError()) {
             decryptionErrorLabelTextView.setVisibility(View.VISIBLE);
             textView.setVisibility(View.VISIBLE);
-            textView.setText("**" + event.getCryptoError().getLocalizedMessage() + "**");
+            textView.setText(event.getCryptoError().getLocalizedMessage());
```

Comment:
Please revert this change. We need to keep consistency with Riot iOS and Riot web

Reply:
Done.